src/views/Orders.vue

```diff
@@ -72,6 +72,8 @@
               <ion-label class="ion-text-wrap">
                 <h1>{{ order.customer.name }}</h1>
                 <p>{{ order.orderName ? order.orderName : order.orderId }}</p>
+                <p v-if="order.pickers && configurePicker && order.pickers.length">{{ $t("Picked by", { pickers: getPickersForOrder(order.pickers) }) }}</p>
```

Comment:
The `if` condition can also be simplified once we will move the logic in action.